### PR TITLE
chore(themes): move to SCSS & sass-lint.

### LINKED ISSUE #384: Replace `stylelint` with a Sass-aware linter
Our `stylelint` check is useless now that we are using Sass. We should use a linter aware of Sass syntax.

https://github.com/brigade/scss-lint should work.

Any other in mind?

### PR DESCRIPTION
**Pull request by redox**
Thanks to http://www.sasstoscss.com/ :)

Fix #384
Ref #290

### CODE REVIEW

**Comment by redox** [comment]
@pixelastic :+1:

File: package.json [hunk 1]
```diff
@@ -54,9 +54,10 @@
     "pretty-bytes": "^2.0.1",
     "raw-loader": "^0.5.1",
     "react-addons-test-utils": "^0.14.0",
+    "sass-lint": "^1.3.2",
+    "scsslint": "0.0.3",
```
Comment (pixelastic) [inline package.json]: Why do we need `scsslint`?

Comment (redox) [inline package.json]: Hmmmm, forgot to remove that one. On it. Good catch!

**Comment by redox** [comment]
Done @pixelastic

**Pull request merged**